src/batch.ts

```diff
@@ -93,3 +94,10 @@ export function aggregate(
     return acc;
   }, results);
 }
+
+/**
+ * The server endpoint for batch.
+ *
+ * @return {String}
+ */
+export const endpoint = endpoints.batch;
```

Comment:
Since `KintoClientBase` has the `endpoint` property, I don't think we need this.